DataAccessLibrary/SqliteDataAccess.cs

```diff
@@ -26,6 +39,18 @@ public async Task<IEnumerable<string>> LoadStaticHotelResources(string group)
             return resources;
         }
 
+        public async Task<IEnumerable<MarketPriceModel>> RetrieveDataFromDb(string table, string column, string value)
+        {
+
+            using IDbConnection connection = new SQLiteConnection(connectionString);
+            IEnumerable<MarketPriceModel> output = await connection.QueryAsync<MarketPriceModel>($"Select {table} From {column} Where {value}", new DynamicParameters());
```

Comment:
Er lidt i tvivl om SQL queryen. Kan man godt bare vælge et table fra en column hvor value? Skal man ikke skrive ind hvad kolonnen hedder i stedet for value, eller tager jeg fejl? 😅

Reply:
Og hvor kommer den her DynamicParameters fra?